### DISCUSSION TITLE
大概什么时候支持 .net 8

### DISCUSSION THREAD

**Discussion by Kakaeex** [dotnetcore/BootstrapBlazor#2423] [Q&A] [answered]
.net8 里blazor模板与 7 比有很大的变化，我尝试将7 模板的代码搬到 8的模板，发现是不能运行的

**Comment by densen2014** [comment]
源码文档工程就是webapp模式，你先参考一下，bb已经支持net8了

**Reply by Kakaeex** [reply]
我试试，thx~

**Comment by ArgoZhang** [comment, marked as answer]
@Kakaeex 拉一下源码看看 还有一个 Auto 分支，这个是 Web App Auto 模式，都支持了
